doc/flame/collision_detection.md

```diff
@@ -120,6 +120,41 @@ other way around. If there are no intersections with the edges on a solid hitbox
 position is instead returned.
 
 
+### Collision order
+
+If a `PositionComponent` collides with more than one other object within a given time step, then
+the `onCollision` callbacks will be called in an essentially random order.  In some cases this can
+be a problem, such as in a bouncing ball game where the trajectory of the ball can differ depending
+on which other object was hit first.  To help resolve this the `collisionsCompletedNotifier`
+listener can be used - this triggers at the end of the collision detection process.
+
+An example of how this might be used is to add a local variable in your `PositionComponent` to save
+the other components with which this one is colliding:
+`List<PositionComponent> collisionComponents = [];`.  The `onCollision` callback is then used to
+save all the other `PositionComponent`s to this list:
```

Comment:
All of these sentences start with two spaces, there should only be one to be consistent with the rest of the docs.

Reply:
Sorry, that's a British English thing, I've changed that.